### PR TITLE
Feat/8080 => Add Dialog for Preview user first login after a signup

### PR DESCRIPTION
**Pull request by morhag90**
Close: #8080

## PR Details

 Add Dialog for Preview user first login after a signup. 

## PR Checklist

- [x] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

### CODE REVIEW

File: packages/amplication-client/src/Project/ProjectPage.scss [hunk 1]
```diff
@@ -1,5 +1,8 @@
 @import "../style/index.scss";
 .project-page {
+  .amp-text {
+    line-height: 24px;
```
Comment (mulygottlieb) [inline packages/amplication-client/src/Project/ProjectPage.scss:4]: This should be in a $ variable according to our guidelines :)

File: packages/amplication-server/src/core/auth/auth.service.ts [hunk 2]
```diff
@@ -763,11 +764,10 @@ export class AuthService {
     let isNew: boolean;
     const existingUser = !!user;
 
+    const isFromPreviewUser =
+      user.account.previewAccountType !== EnumPreviewAccountType.None;
```
Comment (mulygottlieb) [inline packages/amplication-server/src/core/auth/auth.service.ts]:
```suggestion
      user && user.account.previewAccountType !== EnumPreviewAccountType.None;
```

File: packages/amplication-server/src/core/auth/auth.service.ts [hunk 3]
```diff
@@ -763,11 +764,10 @@ export class AuthService {
...
     // to complete the preview account signup, after the user reset the password in Auth0 we need to update the account type and workspace subscription
-    if (
-      user &&
-      user.account.previewAccountType !== EnumPreviewAccountType.None
-    ) {
+    if (user && isFromPreviewUser) {
```
Comment (mulygottlieb) [inline packages/amplication-server/src/core/auth/auth.service.ts]:
```suggestion
    if (isFromPreviewUser) {
```

Comment (morhag90) [reply packages/amplication-server/src/core/auth/auth.service.ts]: @mulygottlieb Already updated :)

**Review by mulygottlieb — APPROVED** [review]

**Pull request merged**